common/org.testeditor.dsl.common/src/org/testeditor/dsl/common/util/classpath/ClasspathUtil.xtend

```diff
@@ -44,22 +44,38 @@ class ClasspathUtil {
 			path = new Path(path.toFile.absolutePath)
 			cpEntry = path.getBuildToolClasspathEntry
 		}
+		if (cpEntry === null) {
+			logger.error("Could not find corresponding classpath entry for path='{}'.", path)
+			return 'com.example'
```

Comment:
default package currently poses a problem we want to adress in a different ticket [OTA-767](http://jira.system.local/browse/OTA-767)